docs/using/troubleshooting.md

```diff
@@ -0,0 +1,307 @@
+# Troubleshooting Common Problems
+
+When troubleshooting, you may see unexpected behaviors or receive an error message. This section provides advice on
+how to identify and fix some of the most commonly encountered issues.
```

Comment:
Please, do not use force pushes.
It makes the review process much worse.
Also, it seems, that you lost the commit which rearranges these lines.

Reply:
so sorry 😢  rebased instead of merging master - promise not to do it again